xbmc/interfaces/python/PythonSwig.cpp.template

```diff
@@ -117,6 +126,8 @@ void doMethod(Node method, MethodType methodType)
     else
       return;
   }
+  else if (method.@name == "contains")
```

Comment:
@jimfcarroll: I have changed the logic. Now you need to add
```
%feature("sequence") Session "contains"
```
into the SWIG interface definition file to make the class `Session` usable as a sequence using the `contains` method for `tp_contains`. The method used for `tp_contains` is automatically excluded from the actual method list.
That should make it more explicit.